.github/workflows/integration-tests.yml

```diff
@@ -41,13 +39,67 @@ jobs:
             echo "zkevm-fixtures-input folder is empty"
             exit 1
           fi
+  
+  stateless-validator:
+    name: Stateless validator guest
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        zkvm: [sp1]
+        action: [execute]
```

Comment:
I tried to include proving, but it is very slow for the CI runner even for proving the empty block.

See [here](https://github.com/eth-act/zkevm-benchmark-workload/actions/runs/16478587156/job/46586663519?pr=121) which got autom. canceled after 6hs.

I'll remove the prove tomorrow... maybe we can think of trying it if we configure a GH runner with a better machine (at min a better CPU/mem, not necessarily need GPU).